src/components/Navbar.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import { siteConfig } from "@/config/SiteConfig";
+
+const Navbar = () => {
+  const pathname = usePathname();
+  const { MainNavBar } = siteConfig;
+
+  return (
+    <>
+      <div className="fixed top-0 inset-x-0 bg-zinc-800 border-b border-zinc-600 py-2">
+        <div className="flex justify-center my-2">
+          <div className="flex gap-4 md:gap-9 text-md font-semibold text-gray-400">
+            {MainNavBar.map((link) => {
+              const isActive = pathname === link.href;
+              return (
+                <Link
+                  className={isActive ? "text-white" : ""}
+                  href={link.href}
+                  key={link.title}
```

Comment:
Personal preference, but I’d use href instead of title. In theory a target URL will be slightly more unique? 🤷‍♂️